modules/tools/testing/build.gradle

```diff
@@ -10,4 +10,7 @@ dependencies {
     compile "org.mockito:mockito-core:${mockitoVersion}"
     compile 'org.hamcrest:hamcrest-library:1.3'
     compile "org.slf4j:slf4j-simple:${slf4jVersion}"
+    testImplementation "com.enonic.xp:lib-content:${version}"
```

Comment:
This is circular dependency. It is simple to avoid if you put js files you want to mock in resources.